src/typeChecker/typeErrorChecker.ts

```diff
@@ -463,6 +571,7 @@ function addTypeDeclarationsToEnvironment(
         }
         const id = node.declarations[0].id as tsEs.Identifier
         const expectedType = getTypeAnnotationType(id.typeAnnotation, context, env)
+        // Type is not checked here as it will be checked in typeCheckAndReturnType
```

Comment:
What's this mean?

Reply:
`addTypeDeclarationsToEnvironment` only deals with hoisting, so there are no type checks done. Will reword this comment accordingly.